_digest/october-2020/choosing-your-major-by-money-or-passion.mdx

```diff
@@ -0,0 +1,32 @@
+  <p>I find the word “education” misleading, and here’s why. When you’re a kid, you have all these dreams and goals. However, from a young age, “education” becomes associated with school and how investing time and money into it will reward you with a bright future. But as our schooling career progresses, we develop a love-hate relationship with the standardized system, and as a result, education itself. We understand that devoting all these years to school can help us achieve a successful career, but we slowly begin to question whether it’s worth the consequential emotional and physical stress.</p>
+  <p>According to Google, the word “education” can be split up into two definitions.</p>
+  
+  <ol>
+    <li> <u><strong><em>“</em></strong><strong><em>The process of receiving or giving systematic instruction, especially at a school or university.”</em></strong> This definition is what most of us think of. It’s the definition that was fed to us by our parents and school. The phrase “systematic instruction” has a negative undertone – as if students don’t have control over our learning, which is what it often feels like. The overpowering pressure to do well in a system built to challenge us has led students to stray away from the principles of schooling, and consequently, education as a whole.</u> </li>
+    <li> <u><strong> </strong><strong><em>“An enlightening experience.”</em></strong> I believe this terminology should be reflected more into our schooling. Regardless of what field you end up pursuing, education can, and should be, an illuminating and uplifting experience.</u> </li>
+  </ol>
+  
+  <p>I understand that often, people don’t have the privilege of getting an education in something they truly enjoy because they don’t have the space for uncertainty. However, I simply want to shed light on another definition of this word that has developed a negative connotation. </p>
+  <p>People tend to neglect taking the arts or humanities because it typically doesn’t make the most money, and that’s a valid point. However, because of this, less and less people are getting educated about the society we live in, and more and more people don’t understand how politics, relationships, and culture work. As a result, there is a giant socio-political divide in our modern society that we experience everyday. Especially with the state the world is in today, it’s more important than ever to be educated about our constantly changing and growing society.</p>
+  <p>“You can fail at what you don’t want, so you might as well take a chance on doing what you love.” – Jim Carrey</p>
+  <p><strong>*This article is in no way telling you what you should and shouldn’t do. It’s just another viewpoint to take note of.</strong></p>
```

Comment:
Can we remove the asterisks here? I think it might mess with the rendering of these files